core/src/main/java/jenkins/security/ApiTokenProperty.java

```diff
@@ -416,22 +420,7 @@ public boolean mustDisplayLegacyApiToken(User propertyOwner) {
         // for Jelly view
         @Restricted(NoExternalUse.class)
         public boolean hasCurrentUserRightToGenerateNewToken(User propertyOwner){
-            if (ADMIN_CAN_GENERATE_NEW_TOKENS && Jenkins.get().hasPermission(Jenkins.ADMINISTER)) {
```

Comment:
CC @Wadeck . what was the reason of this code instead of an existing method? Was it intentional

Reply:
It was intentional: the two methods are in two different classes (regular/internal) and they need to have two separate constants. From my PoV it's a matter of personal style here.